src/tokenizer.rs

```diff
@@ -197,6 +205,10 @@ impl fmt::Display for Token {
             Token::PGSquareRoot => f.write_str("|/"),
             Token::PGCubeRoot => f.write_str("||/"),
             Token::Placeholder(ref s) => write!(f, "{}", s),
+            Token::Arrow => write!(f, "->"),
+            Token::LongArrow => write!(f, "-->"),
```

Comment:
```suggestion
            Token::LongArrow => write!(f, "->>"),
```

Reply:
I wonder why the tests didn't cover this? I think there is a function that verifies that a parsed sql statement is then displayed in the same way

Reply:
We are printing `JsonOperator` not the `Token`. so it escaped :sweat_smile: